tests/firecrackerdriverdata/test_domain.xml

```diff
@@ -0,0 +1,28 @@
+<domain type='firecracker'>
+  <name>firecracker_domain</name>
+  <memory>512000</memory>
+  <currentMemory>512000</currentMemory>
+  <vcpu>1</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc'>hvm</type>
+    <kernel>/home/ec2-user/LV/libvirt/tests/firecrackerdriverdata/hello-vmlinux-test.bin</kernel>
```

Comment:
so anyone looking to run these tests will need to update these paths right?
Maybe add a comment about this.